data/yaml/page-content/asset-tracking.yaml

```diff
@@ -24,13 +24,15 @@ sections:
         type: feature
         content: Use the publishing SDK on the assets you want to track, and the subscribing SDK to observe the location of the assets.
         image: 'trackable-location.svg'
+        link: '/asset-tracking/using-the-sdks'
```

Comment:
Looking at all the usages of `link` here - it occurs to me that we probably don't need to add in a new field here, as `link` replicates `links[0].href`. 

We could cut the complexity of this a little bit by removing `link` and instead offering a helper that takes `links` as an argument and safely returns the first `href` from it. This helper should be put in a location that makes sense to be imported by all the components that use it (i.e. one level up from all of them).